addons/mail/static/src/core/common/settings_model.js

```diff
@@ -30,6 +30,19 @@ export class Settings extends Record {
             return this.channel_notifications === false ? "mentions" : this.channel_notifications;
         },
     });
+    messageSound = Record.attr(true, {
+        compute() {
+            return browser.localStorage.getItem("mail.user_setting.no_message_sound") !== "true";
```

Comment:
This is confusing, I would pick a "positive" name instead, and handling empty value appropriately.

Reply:
The intent to set it that way is so that no customisation = no local storage entry.
By default this is on, so a local storage entry for it means no sound for new message.

Reply:
I understand the intent but it's still confusing. We usually avoid no- fields as they are really confusing to manipulate, having to check !no... for doing something doesn't read well and is prone to errors.

It's really not hard to consider the absence of key as a positive value, and makes the rest easier.

Reply:
I see, you suggest `sound = false` to store, but when undefined this behaves like `sound = true`. Will make this change then